plugins/digitalocean/personal_access_token_test.go

```diff
@@ -22,6 +22,32 @@ func TestPersonalAccessTokenImporter(t *testing.T) {
 				},
 			},
 		},
+		"config file macos": {
+			OS: "darwin",
+			Files: map[string]string{
+				"~/Library/Application Support/doctl/config.yaml": plugintest.LoadFixture(t, "config.yaml"),
+			},
+			ExpectedCandidates: []sdk.ImportCandidate{
+				{
+					Fields: map[sdk.FieldName]string{
+						fieldname.Token: "dop_v1_tr33mpd5m8q9t3ncisqbceydi8dd2n60pl1yiycg97z25fkqffp8j6ycjexample",
+					},
+				},
+			},
+		},
+		"config file linux": {
+			OS: "linux",
+			Files: map[string]string{
+				digitalOceanConfigFileOnLinux(): plugintest.LoadFixture(t, "config.yaml"),
```

Comment:
Because we're executing `digitalOceanConfigFileOnLinux` here in the testcase, this case tests a different scenario based on whether or not `$XDG_HOME` is set on the machine that's running these tests. Can we make the test statically defined instead?

```suggestion
				"~/.config/doctl/config.yaml": plugintest.LoadFixture(t, "config.yaml"),
```